lib/gui/app.js

```diff
@@ -87,22 +91,29 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
   AnalyticsService.logEvent('Application start');
 
   const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates();
-
-  if (!shouldCheckForUpdates || process.env.ETCHER_DISABLE_UPDATES) {
+  const currentVersion = packageJSON.version;
```

Comment:
I didn't go back to `ManifestBindService` given that we're going to get rid if that Angular-specific module as part of the React gradual transition.

> I'm beginning to wonder if it'd make sense to have a .getCurrentVersion function in '../shared/release'?

What about having a different shared module that exposes information about the app such as name, version, description, company, etc? How could we call it? `app-info`?